src/main/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/retry/MessageProducerImpl.java

```diff
@@ -24,9 +23,6 @@
 
     private CHKafkaProducer producer;
 
-    @Autowired
-    private Schema schema;
```

Comment:
Presumably there's a schema or getSchema bean declared somewhere, is that bean used anywhere else now that it is no longer used here?

Reply:
you can now get the schema from the model class  - see https://github.com/companieshouse/chips-rest-interfaces-consumer/pull/29

Reply:
i see what you mean , we can remove the config for this as well